iceoryx_posh/test/moduletests/test_shared_memory_user.cpp

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) 2021 by Robert Bosch GmbH. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Please read the contribution file: https://github.com/eclipse-iceoryx/iceoryx/blob/master/CONTRIBUTING.md